### PR TITLE
bugfix/LEAF-1993/Fixed-Tag-Delete/Import

### PR DESCRIPTION
**Pull request by pete-nerantzinis**
Fixed the problems with the Tags not updating properly and bugging synceservices.

### CODE REVIEW

File: LEAF_Nexus/api/controllers/GroupController.php [hunk 1]
```diff
@@ -67,7 +67,17 @@ public function delete($act)
         $this->index['DELETE']->register('group/[digit]/tag', function ($args) use ($group) {
             try
             {
-                $group->deleteTag((int)$args[0], $group->sanitizeInput($_GET['tag']));
+                return $group->deleteTag((int)$args[0], $_GET['tag'], $_GET['requiresAdmin']);
```
Comment (gsteinhoff) [inline LEAF_Nexus/api/controllers/GroupController.php:70]: Not sure the "requiresAdmin" is needed anymore and tag GET should be run through the sanitizeInput function.

File: LEAF_Nexus/api/controllers/GroupController.php [hunk 2]
```diff
@@ -67,7 +67,17 @@ public function delete($act)
...
+            }
+            catch (Exception $e)
+            {
+                return $e->getMessage();
+            }
+        });
+        $this->index['DELETE']->register('group/[digit]/local/tag', function ($args) use ($group) {
+            try
+            {
+                return $group->deleteLocalTag((int)$args[0], $_GET['tag']);
```
Comment (gsteinhoff) [inline LEAF_Nexus/api/controllers/GroupController.php:80]: Tag GET should be run through the sanitize function

**Review by gsteinhoff — CHANGES_REQUESTED** [review]
Just some sanitizing and cleanup then should be good

File: LEAF_Nexus/api/controllers/GroupController.php (hunk 2 above)
Comment (pete-nerantzinis) [reply LEAF_Nexus/api/controllers/GroupController.php:80]: Good spot thank you

File: LEAF_Nexus/api/controllers/GroupController.php (hunk 1 above)
Comment (pete-nerantzinis) [reply LEAF_Nexus/api/controllers/GroupController.php:70]: Good spot thank you

**Review by Vaceek — APPROVED** [review]
nice

**Review by gsteinhoff — APPROVED** [review]

File: LEAF_Request_Portal/Login.php [hunk 3]
```diff
@@ -456,7 +456,8 @@ public function getMembership()
         // incorporate groups from local DB
         $vars = array(':userName' => $this->userID);
         $res = $this->userDB->prepared_query('SELECT * FROM users
-												WHERE userID = :userName', $vars);
+												WHERE userID = :userName
```
Comment (Pelentan) [inline LEAF_Request_Portal/Login.php:459]: Please put this all on one line.

**Review by Pelentan — CHANGES_REQUESTED** [review]

**Review by Pelentan — APPROVED** [review]
Looks good.

**Pull request merged**